### ISSUE TITLE
Color Bulb status update only on double button press

### ISSUE THREAD

**Issue by darkaeden** [iklein99/homebridge-smartthings#278]
I have four color bulbs connected to Smartthings connected via homebridge-smartthings to Homekit Home app.

Whenever i press any of such bulb in Apple Home to ON it switches to ON (physical world) but the button goes back to showing OFF. Only if i press it again, it shows ON without giving another ON command to the bulb (physical world).

Same for OFF: the first press turn the Bulb OFF for real, but only a second press will then show in Home app that the bulb is OFF.

Basically, i have to press button in home twice to properly show real status.

Plugin:
Last version

Homebridge
Virtualbox in Win11 Pro

Node
Last version
